zubhub_frontend/zubhub/src/views/projects/Projects.jsx

```diff
@@ -164,35 +168,48 @@ function Projects(props) {
                   </Grid>
                 ))}
               </Grid>
-              <div aria-label={t('projects.ariaLabels.prevNxtButtons')} className={classes.buttonGroupStyle}>
+              <div aria-label={t('projects.ariaLabels.prevNxtButtons')} className={classes.buttonGroupStyleThree}>
                 {prev_page ? (
                   <CustomButton
                     className={clsx(classes.floatLeft, classes.buttonGroupStyleAlternative)}
                     size="large"
                     startIcon={<NavigateBeforeIcon />}
-                    onClick={(e, page = prev_page.split('?')[1]) => {
+                    onClick={(e, page = prev_page.substring('=', "&")) => {
                       handleSetState({ loading: true });
                       handleSetState(fetchPage(page, props));
                     }}
                     primaryButtonStyle
                   >
                     {t('projects.prev')}
                   </CustomButton>
-                ) : null}
+                ) : <CustomButton className={clsx(classes.visibilityNone)}></CustomButton>}
+                    <Pagination
```

Comment:
Can we align this horizontally centered with the Next and Prev button. We can see in screenshots that this is not